### PR TITLE
Details

### PR DESCRIPTION
**Pull request by srobert99**
#### Description
Created details screen, the views are scalabe and it works properly on phone/tablet screens.

#### [Task](task_url)
https://trello.com/c/zdOYA1fX/39-android-freaks-screen-details-screen

### CODE REVIEW

**Comment by srobert99** [comment]
[Android: Freaks Screen - Details Screen](https://trello.com/c/zdOYA1fX/39-android-freaks-screen-details-screen)

**Comment by srobert99** [comment]
[Android: Freaks Screen - Details Screen](https://trello.com/c/zdOYA1fX/39-android-freaks-screen-details-screen)

File: build.gradle.kts [hunk 1]
```diff
@@ -12,6 +12,7 @@ allprojects {
 }
 
 buildscript {
+    val kotlin_version by extra("1.5.21")
```
Comment (AlexandraDamaschin) [inline build.gradle.kts]: Why do you add this?

File: commons/ui/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest package="com.agilefreaks.freaks_catalog.commons.ui">
 
+
```
Comment (AlexandraDamaschin) [inline commons/ui/src/main/AndroidManifest.xml]: Why do you want to add this extra empty line?

File: features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreakDetailsFragment.kt [hunk 3]
```diff
@@ -4,21 +4,44 @@ import android.os.Bundle
...
+        setViews()
         return viewBinding.root
     }
+
+    private fun setViews() {
+        arguments?.let {
+            val freak = it.getSerializable("freak") as Freak
+            activity
+            viewBinding.descriptionTV.text = freak.description
+            viewBinding.skillsTV.text =
```
Comment (AlexandraDamaschin) [inline features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreakDetailsFragment.kt]: What is TV?

File: features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreakDetailsFragment.kt [hunk 4]
```diff
@@ -4,21 +4,44 @@ import android.os.Bundle
...
+            viewBinding.projectsTV.text =
+                getString(R.string.projects_template, freak.projects.joinToString(", "))
+            viewBinding.titleTV.text = getString(R.string.title_template, freak.role, freak.norm)
+            viewBinding.levelTV.text = getString(R.string.level_project, freak.level)
+            initBar(freak.firstName,freak.lastName)
+        }
+    }
+
+    fun initBar(firstName:String, lastName:String) {
+        val mainActivityTB = (activity as AppCompatActivity).supportActionBar
```
Comment (AlexandraDamaschin) [inline features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreakDetailsFragment.kt]: What means TB ?

File: features/freaks/src/main/res/drawable/profile_pick_background.xml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <corners android:radius="10dp" />
+
```
Comment (AlexandraDamaschin) [inline features/freaks/src/main/res/drawable/profile_pick_background.xml:5]: Extra empty space

File: features/freaks/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -1,3 +1,24 @@
 <resources>
     <string name="freaks_title">Freaks screen</string>
+    <string name="skills_btn">Skills</string>
+    <string name="projects_btn">Projects</string>
+    <string name="resetBtn">Reset</string>
+    <string name="androidCheckBox">Android</string>
+    <string name="iosCheckBox">iOS</string>
+    <string name="cevaCheckBox">Ceva</string>
```
Comment (AlexandraDamaschin) [inline features/freaks/src/main/res/values/strings.xml]: Love this ceva/altceva code

File: features/freaks/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -1,3 +1,24 @@
...
+    <string name="description_template">
+        Lorem ipsum dolor sit amet, consecte adipiscing elit. Aenean commodo eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis part montes, nascetur ridiculus mus.
+    </string>
+    <string name="title_template">%1$s - %2$s</string>
+    <string name="skills_template">Skills: %1$s</string>
+    <string name="projects_template">Projects: %1$s</string>
+    <string name="level_project">Level: %1$s</string>
+    <string name="profile_image">profile_image</string>
+    <string name="name_template">%1$s %2$s</string>
+    <string name="back_button">back button</string>
```
Comment (AlexandraDamaschin) [inline features/freaks/src/main/res/values/strings.xml:16]: Do we really use all these strings?

**Review by AlexandraDamaschin — CHANGES_REQUESTED** [review]
Some improvements.

**Review by AlexandraDamaschin — APPROVED** [review]
Does this improve software quality? Yes

**Pull request merged**